source/tutorial/6_inputsoutputs.rst

```diff
@@ -3,14 +3,14 @@ Narrative User Interface
 
 Apps on KBase, such as genome assemblers and annotators that run on narrative pages, are created using the KBase SDK.
 The user interface (UI) that users see in narratives and in the app catalog are defined in two files: 
-``spec.json`` and ``display.yaml``. These two files are in a directory called ``ui/narrative/methods/<app_name>``
-where ``<app_name>`` is the name of the function that you want to turn into an app. In the example module,
+``spec.json`` and ``display.yaml``. These two files are in a directory called ``ui/narrative/methods/run_{uid}{module_name}``
+where ``{module_name}`` is the name of the function that you want to turn into an app. In the example module,
```

Comment:
So `run_{uid}{module_name}` is the name of the function/app, while `{uid}{module_name}` is the name of the module. 

In kbase terminology:
Module - whole repo, can have many functions
Function - a single function in a module. May or may not have UI config for it. Synonymous with method.
App - a single function with UI. Lives inside a module. Can be run in a narrative.

So module should not be a synonym with method, function, or app.

Reply:
Got it, thanks. Pretty sure I'm straight on this now. Check out line 13 in the '6_inputsoutputs.rst' file - is that information consistent with your note here?